### PR TITLE
Add max_fps option to the FIM documentation.

### LINKED ISSUE #3217: Add max_fps option for FIM
Hi team

After developing the issue https://github.com/wazuh/wazuh/issues/5645 a new option has been added for FIM and it should be added to the documentation. 

This option can set the maximum number of files that FIM scans per second. By default, its value is 0 (no limit) and it can take any positive integer.

Kind regards

### PR DESCRIPTION
**Pull request by antoniomanuelfr**
## Description
Hi team. 
This PR adds a new option for FIM: `max_fps`. This option is described in #3217. 

Closes #3217.

## Checks
- [X] It compiles without warnings.
- [x] Spelling and grammar. 
- [X] Used impersonal speech. 
- [X] Used uppercase only on nouns.

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 1]
```diff
@@ -442,6 +443,28 @@ Example:
...
 
 
+.. _reference_ossec_syscheck_max_files_per_second:
+
+max_files_per_second
+--------------------
+
+.. versionadded:: 4.2.0
+
+Sets the maximum number of files scanned per second. It only applies to the scheduled scan. If this option is set to 0, there will be no limit on the number of files scanned per second.
```
Comment (Molter73) [inline source/user-manual/reference/ossec-conf/syscheck.rst:453]:
```suggestion
Sets the maximum number of files scanned per second. If this option is set to 0, there will be no limit on the number of files scanned per second.
```

**Review by Molter73 — CHANGES_REQUESTED** [review]

**Review by Molter73 — APPROVED** [review]
LGTM!

File: source/user-manual/reference/ossec-conf/syscheck.rst [hunk 2]
```diff
@@ -442,6 +443,28 @@ Example:
...
+--------------------
+
+.. versionadded:: 4.2.0
+
+Sets the maximum number of files scanned per second. If this option is set to 0, there will be no limit on the number of files scanned per second.
+
++--------------------+---------------------------------------------------------+
+| **Default value**  | 0                                                       |
++--------------------+---------------------------------------------------------+
+| **Allowed values** | Integer positive number. 0 mean no limit.               |
```
Comment (davidfmiranda) [inline source/user-manual/reference/ossec-conf/syscheck.rst:458]:
```suggestion
| **Allowed values** | Integer positive number. 0 means no limit.              |
```

Comment (davidfmiranda) [inline source/user-manual/reference/ossec-conf/syscheck.rst:458]:
```suggestion
| **Allowed values** | Integer positive number. 0 means no limit.              |
```

**Review by davidfmiranda — CHANGES_REQUESTED** [review]
Hello @antoniomanuelfr, I've reviewed your PR, and looks good to me. Nevertheless, there is a small typo to be fixed.

Regards,

David

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**